### PR TITLE
Add script to lookup userids in identitystore.

### PR DESCRIPTION
**Pull request by abuggia**

### CODE REVIEW

**Review by druiz — APPROVED** [review]
That's cool! Might want to mention that they would need to be in an `aws-vault` session and maybe add some AWS account id informations to the output, so the user is clear on which account his identities are coming from.

File: sym_community_scripts/identitystore_userids.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    def _list_users(self, **filters):
+
+        attrval_filters = [{"AttributePath": k, "AttributeValue": v} for k, v in filters.items()]
+
+        return self.identitystore.list_users(
+            IdentityStoreId=self.identitystore_id, MaxResults=50, Filters=attrval_filters
+        )
+
+    def get_userid(self, username):
+        response = self._list_users(UserName=username)
```
Comment (jon918) [inline sym_community_scripts/identitystore_userids.py:20]: Looks like there is a filter option to find a specific user: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/identitystore.html#IdentityStore.Client.list_users

If that doesn't work, then i think this needs to account for pagination.

**Review by jon918 — CHANGES_REQUESTED** [review]

File: sym_community_scripts/identitystore_userids.py (hunk 1 above)
Comment (jon918) [reply sym_community_scripts/identitystore_userids.py:20]: I think that right now it is possible to only have one identity store per aws acct, so if not specified could look up the default identity store id.

File: sym_community_scripts/identitystore_userids.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import boto3
+import argparse
+import sys
+import botocore
+
+
+class IdentityStore:
```
Comment (yasyf) [inline sym_community_scripts/identitystore_userids.py:7]: Should inherit from the shared base class and use the helpers, if possible

**Pull request merged**